infrastructure/cdn-in-a-box/enroller/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN go clean && go get -v && go build
 
 FROM debian:stretch
 
-RUN apt-get update && apt-get install -y netcat curl dnsutils net-tools && apt-get clean
+RUN apt-get update && apt-get --no-install-recommends install -y netcat curl dnsutils net-tools && apt-get clean
```

Comment:
See above—I would prefer disabling `APT::Install-Recommends` and `APT::Install-Suggests` in the APT configuration to passing `--no-install-recommends` for every apt or apt-get command.